doc/releases/release-notes-1.14.rst

```diff
@@ -326,6 +326,7 @@ Build and Infrastructure
 
 * Added support for out of tree architectures
 * `BabbleSim`_ has been integrated in Zephyr's CI system.
+* Added support for Zephyr modules / multi-repo
```

Comment:
```suggestion
* Added west tool for multiple repository management and out of tree zephyr modules
* Build, flash, and debug targets now require west
```

Reply:
yep, good point about the `build, flash, etc` commands.
But about `west`, then `west` itself is not part of Zephyr repo, that's why I didn't mention it, but will do now.

But will keep zephyr modules and `west` on separate lines.

Reply:
done